src/app/legacy/psammead/psammead-navigation/src/ScrollableNavigation/index.jsx

```diff
@@ -16,6 +16,15 @@ const hexToRGB = (hex, alpha = 1) => {
   return `rgba(${r}, ${g}, ${b}, ${alpha})`;
 };
 
+const focusIndicatorThickness = '0.1875rem'; // 3px
```

Comment:
**nitpick**: I am seeing this in a few different places. Is there perhaps a common place we could declare this value, and then just import it where we need it?

Reply:
Good idea! I have added it as a Constant into the focusIndicator.ts file and imported into the relevant files. I have also refactored the focusIndicator.ts file to use the constant rather than the PixelsToRem function.